index.html

```
@@ -1,12 +1,65 @@
+              <button class="primary detail">자세히 보기</button>
+            </div>
+            <img src="./images/woowacourse_logo.png" alt="영화 이미지" />
+          </div> -->
+        </div>
+      </header>
+      <div class="container">
+        <main>
+          <section>
+            <p class="page-title">지금 인기 있는 영화</p>
+            <div class="search-error-container" hidden>
+              <img src="" alt="" />
```

HTMLHint (1.9.2)

[error] The attribute [ src ] of the tag [ img ] must have a value.

(src-not-empty)